test/DynamoCoreWpfTests/SerializationTests.cs

```diff
@@ -464,8 +465,13 @@ public void JSONisSameBeforeAndAfterSaveWithDummyNodes()
             // Ignoring the ExtensionWorkspaceData property as this is added after the re-save,
             // this will cause a difference between jobject1 and jobject2 if it is not ignored.
             // Same thing goes for the Linting property...
+            // We also need to ignore the new IsCollapsed property on ViewModelBase
```

Comment:
hmm can we add JsonIgnore on `ViewModelBase`?

Reply:
will still need this property serialized, but as this property isn't there when we open the graph but gets applied after save the test will fail